gpt_engineer/main.py

```diff
@@ -20,7 +25,7 @@ def main(
     project_path: str = typer.Argument("example", help="path"),
     delete_existing: bool = typer.Argument(False, help="delete existing files"),
     model: str = "gpt-4",
-    temperature: float = 0.1,
+    temperature: float = typer.Option(0.0, "--temperature", "-t", help="use 0.1 to add some variability"),
```

Comment:
If you want to change the temperature you can do so by passing the proper parameter when running the command.

```
$ gpt-engineer --help

Usage: gpt-engineer [OPTIONS] [PROJECT_PATH] [DELETE_EXISTING]

Arguments:
  [PROJECT_PATH]     path  [default: example]
  [DELETE_EXISTING]  delete existing files  [default: False]

Options:
  --model TEXT                    [default: gpt-4]
  --temperature FLOAT             [default: 0.1]
  -s, --steps [default|benchmark|simple|tdd|tdd+|clarify|respec|execute_only|use_feedback]
                                  decide which steps to run  [default:
                                  Config.DEFAULT]
  -v, --verbose
  --run-prefix TEXT               run prefix, if you want to run multiple
                                  variants of the same project and later
                                  compare them
  --install-completion [bash|zsh|fish|powershell|pwsh]
                                  Install completion for the specified shell.
  --show-completion [bash|zsh|fish|powershell|pwsh]
                                  Show completion for the specified shell, to
                                  copy it or customize the installation.
  --help                          Show this message and exit.
  ```